### PR TITLE
Edits to deploy-chat-teams-channels-microsoft-teams-landing-page

### PR DESCRIPTION
**Pull request by jdmartinez36**
Edited description to add keywords, changed first H2, added requested keywords in several instances.

### CODE REVIEW

**Comment by jdmartinez36** [comment]
@DaniHalfin

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/deploy-chat-teams-channels-microsoft-teams-landing-page.md [hunk 1]
```diff
@@ -19,26 +19,27 @@ ms.custom:
...
 
 *New in November 2019*
- - You can now [use Advisor for Teams (preview) to help you roll out Microsoft Teams](use-advisor-teams-roll-out.md). Advisor for Teams (preview) walks you through your Teams rollout. It assesses your Office 365 environment and identifies the most common configurations that you may need to update or modify before you can successfully roll out Teams.
+ - You can now [use Advisor for Teams (preview) to help you roll out Microsoft Teams](use-advisor-teams-roll-out.md). Advisor for Teams (preview) walks you through your Teams rollout. It assesses your Office 365 environment and identifies the most common configurations that you may need to update or modify before you can successfully roll out the Teams app.
  - [Microsoft Teams Essentials for IT YouTube channel](https://aka.ms/MicrosoftTeamsforIT), including short (8-10 minute) videos that show you how to roll out, configure, and manage Teams.
 
 > [!TIP]
 > We recommend that you include our featured apps -- such as Planner -- in your initial Teams rollout. Add other [apps, bots, and connectors](deploy-apps-microsoft-teams-landing-page.md) as you drive Teams adoption.
```
Comment (JohanFreelancer9) [inline Teams/deploy-chat-teams-channels-microsoft-teams-landing-page.md:38]:
```suggestion
> We recommend that you include our featured apps, such as Planner, in your initial Teams rollout. Add other [apps, bots, and connectors](deploy-apps-microsoft-teams-landing-page.md) as you drive Teams adoption.
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@DaniHalfin Hi! Is this PR going to be closed? Thanks!

**Pull request merged**